extensions/braze/braze-currents-users-event.schema.json

```diff
@@ -240,30 +445,143 @@
+                  "description": "Text of button pressed by the user."
+                }
+              }
+            },
+            "braze:pushNotification": {
+              "title": "Push Notification",
+              "type": "object",
+              "properties": {
+                "braze:buttonString": {
+                  "title": "Button String",
+                  "type": "string",
+                  "description": "Identifier (button_string) of the push notification button clicked. null if not from a button click."
+                },
+                "braze:buttonActionType": {
+                  "title": "Button Action Type",
+                  "type": "string",
+                  "description": "Action type of the push notification button. one of [uri, deep_link, none, close]. null if not from a button click.",
+                  "enum": [
+                    "URI",
+                    "DEEP_LINK",
+                    "NONE",
+                    "CLOSE"
+                  ],
+                  "meta:enum": {
+                    "URI": "URI",
+                    "DEEP_LINK": "Deep Link",
+                    "NONE": "None",
+                    "CLOSE": "Close"
+                  }
+                }
+              }
+            },
+            "braze:in-appMessage": {
+              "title": "In-App Message",
+              "type": "object",
+              "properties": {
+                "braze:version": {
+                  "title": "Version",
+                  "type": "string",
+                  "description": "Which version of in app message, legacy or triggered."
+                },
+                "braze:buttonID": {
+                  "title": "Button ID",
+                  "type": "string",
+                  "description": "ID of the button clicked, if this click represents a click on a button."
+                }
+              }
+            },
+            "braze:error": {
+              "title": "Error",
+              "type": "object",
+              "properties": {
+                "braze:error": {
+                  "title": "Error",
+                  "type": "string",
+                  "description": "Error name."
+                },
+                "braze:providerErrorCode": {
+                  "title": "Provider Error Code",
+                  "type": "string",
+                  "description": "Error code from WhatsApp."
+                },
+                "braze:providerErrorTitle": {
+                  "title": "Provider Error Title",
+                  "type": "string",
+                  "description": "Description of failure from WhatsApp."
                 }
               }
             }
           }
         },
+        "braze:session": {
+          "title": "Session",
+          "type": "object",
+          "properties": {
+            "braze:duration": {
+              "title": "Duration",
+              "type": "number",
+              "description": "Duration of the session."
```

Comment:
Is there an expected time unit? Then maybe it should be included in the description (like days/hours/minutes/seconds/milliseconds). You could also consider changing the field name to say `durationMilliseconds` or `durationSeconds` etc.

Reply:
That's a good idea. Changing the field name is complicated on our end, but I'll update the documentation once I have an answer.